dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/file-manage/docker-compose.yaml

```diff
@@ -35,22 +35,51 @@ services:
     volumes:
       - ./common.properties:/opt/dolphinscheduler/conf/common.properties
     depends_on:
-      hdfs:
+      s3:
         condition: service_healthy
-  hdfs:
-    image: mdouchement/hdfs:latest
-    hostname: hdfs
+      mc:
+        condition: service_healthy
+  s3:
+    image: minio/minio:latest
+    hostname: s3
     tty: true
     stdin_open: true
-    expose:
-      - 8020
+    command: server /data --console-address ":9001"
+    ports:
+      - 9000:9000
     networks:
       - e2e
+    environment:
+      MINIO_ROOT_USER: accessKey123
+      MINIO_ROOT_PASSWORD: secretKey123
     healthcheck:
-      test: [ "CMD", "curl", "http://localhost:50070" ]
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
       interval: 5s
       timeout: 120s
       retries: 120
-
+  mc:
+    image: minio/mc:latest
+    entrypoint: ""
+    hostname: mc
+    tty: true
+    stdin_open: true
+    networks:
+      - e2e
+    command: bash -c '
+      mc alias set s3 http://s3:9000 accessKey123 secretKey123
+      && mc mb s3/dolphinscheduler && tail -f /dev/null'
```

Comment:
Do we need to keep this running? I think we can let it exit normally without `tail -f /dev/null`

Reply:
This is a very lightweight container that does not take up too much resources. Using tail - F to make the container pass the health check(That means the s3 bucket have been created.). So that the standalone server can running test after this. Is there a better way to solve this?

Reply:
If you do this, even if the command `bash -c xxxxxxx` failed I think it's still considered as healthy because your health check command is `echo 1`.

You don't need health check for Init Containers, you can use something like 

```yaml
  dependes_on:
      mc:
         condition: service_completed_successfully
```

Reply:
```
services.dolphinscheduler.depends_on.mc.condition contains "service_completed_successfully", which is an invalid type, it should be a service_started, or a service_healthy
```

Support for service_completed_successfully has been removed until added in the latest version of docker-compose 1.29.
However, the built-in docker compose version of Ubuntu 20.04 is only 1.25 at most. It seems that we can only use
service_healthy or service_started...

https://github.com/docker/compose/issues/8154